### PR TITLE
Update/ordering according to difficulty

### PR DESCRIPTION
**Pull request by tarang1998**

### CODE REVIEW

**Comment by tarang1998** [comment]
@imkgarg 
Could you please review this 

Did this just as an example

Should I arrange the rest of the sections in a similar way ??

File: Coding.md [hunk 1]
```diff
@@ -26,26 +26,30 @@
...
 - https://www.geeksforgeeks.org/mathematical-algorithms/?ref=ghm#gcd
+- https://www.hackerearth.com/practice/notes/euclid-algorithm-made-easy-1/
 - https://www.geeksforgeeks.org/mathematical-algorithms/?ref=ghm#prime
 - https://www.geeksforgeeks.org/mathematical-algorithms/?ref=ghm#factors
 - https://www.geeksforgeeks.org/sieve-of-eratosthenes/
 - https://www.techiedelight.com/power-function-implementation-recursive-iterative/
 
 ### Questions
 
-- https://www.hackerearth.com/practice/notes/euclid-algorithm-made-easy-1/
```
Comment (tarang1998) [inline Coding.md:36]: Added this to the theory section

**Comment by imkgarg** [comment]
@tarang1998 is there no hard level question in the maths section right now?

**Comment by tarang1998** [comment]
@imkgarg 
Nope 
I didn't find any question that was labelled as hard

**Comment by tarang1998** [comment]
I'll sort the rest of the sections in a similar way then

**Pull request merged**